jcommune-model/src/main/resources/org/jtalks/jcommune/migrations/V65__Add_column_DELETED_to_Post_Comment.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE POST_COMMENT ADD(DELETED TINYINT(1) NOT NULL DEFAULT 0);
```

Comment:
We discussed with @ctapobep and decided to use datetime of deletion instead of flag.